plugins/azure-devops-backend/src/api/AzureDevOpsApi.ts

```diff
@@ -393,6 +396,36 @@ export class AzureDevOpsApi {
 
     return buildRuns;
   }
+
+  public async getReadme(
+    host: string,
+    organization: string,
+    projectName: string,
+    repoName: string,
+  ): Promise<{
+    url: string;
+    content: string;
+  }> {
+    const getFileContent = async (
```

Comment:
maybe not worth making this separate function here; the amount of boilerplate syntax is greater than actual effective lines :)